templates/prometheusremotewriterspec.go

```diff
@@ -0,0 +1,56 @@
+package templates
+
+import (
+	"strings"
+
+	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+var metrics = []string{
+	"CephMdsMissingReplicas",
+	"CephMgrIsAbsent",
+	"CephMgrIsMissingReplicas",
+	"CephNodeDown",
+	"CephClusterErrorState",
+	"CephClusterWarningState",
+	"CephOSDVersionMismatch",
+	"CephMonVersionMismatch",
+	"CephOSDFlapping",
+	"CephOSDDiskNotResponding",
+	"CephOSDDiskUnavailable",
+	"CephDataRecoveryTakingTooLong",
+	"CephPGRepairTakingTooLong",
+	"CephMonQuorumAtRisk",
+	"CephMonQuorumLost",
+	"job:ceph_versions_running:count",
+	"job:ceph_pools_iops_bytes:total",
+	"job:ceph_pools_iops:total",
+	"job:kube_pv:count",
+	"job:ceph_osd_metadata:count",
+	"ceph_health_status",
+	"ceph_cluster_total_used_raw_bytes",
+	"ceph_cluster_total_bytes",
+	"cluster:kubelet_volume_stats_used_bytes:provisioner:sum",
+	"cluster:kube_persistentvolumeclaim_resource_requests_storage_bytes:provisioner:sum",
+}
+var PrometheusRemoteWriteSpecTemplate = &promv1.RemoteWriteSpec{
+	OAuth2: &promv1.OAuth2{
+		ClientSecret: corev1.SecretKeySelector{
+			LocalObjectReference: corev1.LocalObjectReference{},
+		},
+		ClientID: promv1.SecretOrConfigMap{
+			Secret: &corev1.SecretKeySelector{
+				LocalObjectReference: corev1.LocalObjectReference{},
+			},
+		},
+		EndpointParams: map[string]string{},
+	},
+	WriteRelabelConfigs: []promv1.RelabelConfig{
+		{
+			SourceLabels: []string{"__name__"},
+			Regex:        strings.Join(metrics[:], "|"),
+			Action:       "keep",
+		},
+	},
+}
```

Comment:
My prev review requested for this to be rolled into the prom template (https://github.com/red-hat-storage/ocs-osd-deployer/pull/208#discussion_r898093495) and the conversation was marked as done.

Is there a reason why is this still its own thing?

Reply:
Sorry, moved it to prom template